posthog/api/team.py

```diff
@@ -316,6 +341,17 @@ def update(self, instance: Team, validated_data: Dict[str, Any]) -> Team:
         if "timezone" in validated_data and validated_data["timezone"] != instance.timezone:
             self._handle_timezone_update(instance)
 
+        if (
+            "session_replay_config" in validated_data
+            and validated_data["session_replay_config"] is not None
+            and instance.session_replay_config is not None
+        ):
+            # don't change keys that are not provided, that would be confusing
```

Comment:
This is only true for the top level keys right? If my config is `{ ai_config: preferred_events: [...], opt_in: true }` and my validated data is `{ ai_config: opt_in: false }` we would override the nested config right?

I think we currently protect against this on the frontend by always including the whole object